Week2/homework/js-exercises/2_ EvenOddReporter.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
+
+for (let i=0; i<=20; i++) {
+
+    if (i%2 === 0) {
+        console.log("The number " + i.toString() + " is even")
+    } else {
+        console.log("The number " + i.toString() + " is odd")
+    }
+
+}; 
```

Comment:
👍 for logic but some unnecessary new lines here which can be avoided for better code styling purposes 🙂 

But don't worry, you will learn about it in the future modules.